src/Components/ReviewList.jsx

```diff
@@ -0,0 +1,33 @@
+import { fetchReviews } from "../api";
+import { useEffect, useState } from "react";
+import ReviewCard from "./ReviewCard";
+
+function ReviewList() {
+    const [currentReviews, setCurrentReviews] = useState([])
+    const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true)
```

Comment:
When the page loads the default state is true, you don't need to add this line. Even when it gets set to false on line 13, when the page re renders the default is back to true.

Reply:
Good point, thank you!